src/esp/assets/RenderAssetInstanceCreationInfo.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include "Asset.h"
+
+#include "esp/scene/SceneNode.h"
+
+#include "Magnum/Resource.h"
+
+#include <Corrade/Containers/EnumSet.h>
+#include <Corrade/Containers/Optional.h>
+
+namespace esp {
+namespace assets {
+
+// parameters to control how a render asset instance is created
+struct RenderAssetInstanceCreationInfo {
+  enum class Flag : unsigned int {
+    IsStatic = 1 << 0,  // inform the renderer that this instance won't be moved
+    IsRGBD = 1 << 1,    // use in RGB and depth observations
+    IsSemantic = 1 << 2  // use in semantic observations
+  };
+  typedef Corrade::Containers::EnumSet<Flag> Flags;
+
+  RenderAssetInstanceCreationInfo(
+      const std::string& _filepath,
+      const Corrade::Containers::Optional<Magnum::Vector3>& _scale,
+      bool isStatic,
```

Comment:
Because you have defined Flags, these 3 booleans can be replaced by a 'Flags flags`, aren't they?
That is the purpose to have the Flags introduced. Did I miss anything?